### PR TITLE
add a connect siri to gtfs notebook

### PR DESCRIPTION
**Pull request by assafpi**
added the initial notebook, will be modified later

### CODE REVIEW

**Comment by cjer** [comment]
@EyalBerger Can you take a look at this?

File: notebooks/Connect_siri_to_gtfs.ipynb [hunk 1]
```diff
@@ -0,0 +1,6639 @@
...
+       "      <td>32.073338</td>\n",
+       "      <td>34.766151</td>\n",
+       "      <td>18</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>7091577</th>\n",
+       "      <td>104537</td>\n",
+       "      <td>32.073349</td>\n",
+       "      <td>34.766179</td>\n",
+       "
```
Comment (EyalBerger) [inline notebooks/Connect_siri_to_gtfs.ipynb]: I would use Pandas for filtering.
For example: 
```
start_date = pd.datetime(2019,5,22).date()
end_date = pd.datetime(2019,5,23).date()
siri_data[(pd.to_datetime(siri_data['date']).dt.date>=start_date)&
         (pd.to_datetime(siri_data['date']).dt.date<=end_date)].head()
```

Comment (EyalBerger) [inline notebooks/Connect_siri_to_gtfs.ipynb]: It is better to use `head` for displaying a big DataFrame

**Comment by cjer** [comment]
So what is our intention with this notebook? What issue does this solve?

**Comment by cjer** [comment]
I am closing this for now. Lets keep discussions prior to working versions in Issues.

**Pull request closed without merging**